meiga/result.py

```diff
@@ -219,11 +226,11 @@ def handle(
 
         return self
 
-    def map(self, transform: Callable) -> None:
+    def map(self, mapper: Callable) -> None:
```

Comment:
Pretty sure one cannot use `Callable` without type arguments in `[]`. Maybe update it to `Callable[..., Any]` while we're here?

Reply:
Ohh thank you for this advice.